app/backend/src/clj/teet/meeting/meeting_db.clj

```diff
@@ -254,3 +254,26 @@
            db (:db/id user) meeting-id)
       not-empty
       boolean))
+
+(defn export-meeting
+  "Fetch information required for export meeting PDF"
+  [db user id]
+  (link-db/fetch-links
+    db user {} #(or (contains? % :meeting.agenda/topic)
+                    (contains? % :meeting.decision/body))
+           (d/pull db '[:db/id
```

Comment:
formatting looks weird here, the `(d/pull ...` should start at the same column as db on line 262